main.nf

```diff
@@ -193,10 +194,16 @@ ch_output_docs = file("$baseDir/docs/output.md", checkIfExists: true)
 ch_output_docs_images = file("$baseDir/docs/images/", checkIfExists: true)
 
 
-// Validate inputs
-if (!(params.spectra || params.sdrf) || (params.spectra && params.sdrf))
-{
-  log.error "EITHER spectra data OR SDRF needs to be provided. Make sure you have used either of those options."; exit 1
+params.sdrf = ""
+params.spectra = ""
+
+// Validate input
+if (params.input.endsWith("sdrf") || params.input.endsWith("SDRF")) {
```

Comment:
I would suggest to use 
https://www.tutorialspoint.com/groovy/groovy_tolowercase.htm
and then compare against the lowercase version in all cases.

Prettier and all cases, because weird people do weird things.
I would not be surprised to see someone use .Raw or MZML

Reply:
Yep make sense.